pxr/imaging/plugin/hdRpr/rprApi.cpp

```diff
@@ -1430,6 +1431,19 @@ class HdRprApiImpl {
             return;
         }
 
+        if (!m_rprSceneExportPath.empty()) {
+            std::string exportPath;
+            {
+                std::lock_guard<std::mutex> lock(m_rprSceneExportPathMutex);
+                std::swap(m_rprSceneExportPath, exportPath);
+            }
+            if (!exportPath.empty()) {
+                auto rprContextHandle = rpr::GetRprObject(m_rprContext.get());
```

Comment:
I've marked this PR as "draft" because of this line. This line will compile only when https://github.com/Radeon-Pro/RadeonProRenderAPI/pull/68 is merged